src/main/java/com/worktracker/controller/ReportController.java

```diff
@@ -0,0 +1,36 @@
+package com.worktracker.controller;
+
+import com.worktracker.model.dto.ReportByProjectResponseDTO;
+import com.worktracker.model.dto.ReportByUserResponseDTO;
+import com.worktracker.service.ReportService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.time.LocalDate;
+import java.util.List;
+
+@RestController
+@RequestMapping("/reports")
+public class ReportController {
+
+    private final ReportService reportService;
+
+    @Autowired
+    public ReportController(ReportService reportService) {
+        this.reportService = reportService;
+    }
+
+    @GetMapping("/project/{id}")
```

Comment:
Lets follow the convention for the other endpoints and make it plural like so `projects`